src/openrct2-ui/windows/NewRide.cpp

```diff
@@ -67,8 +67,8 @@ namespace OpenRCT2::Ui::Windows
         RIDE_TYPE_SIDE_FRICTION_ROLLER_COASTER,
         RIDE_TYPE_VIRGINIA_REEL,
         RIDE_TYPE_REVERSER_ROLLER_COASTER,
-        RIDE_TYPE_CLASSIC_WOODEN_ROLLER_COASTER,
         RIDE_TYPE_WOODEN_ROLLER_COASTER,
+        RIDE_TYPE_CLASSIC_WOODEN_ROLLER_COASTER,
```

Comment:
Maybe we should sort the "classic" types before their RCT2 after all. After all, it sort-of represents an older coaster type in much the same way that the side-friction, virginia reel and reverser do.

Reply:
I'm a bit unsure, because on the other hand you have the classic mini coaster which is more advanced than the junior so it seems weird to put it before the less capable junior coaster. The classic stand-up is also kind of like this thanks to it having powered launches & a higher max build height so it would be weird to put it before the RCT2 stand-up imo. Although it does lack helix pieces so there is that against it at least.

Reply:
In that case, let’s just put the Classic Wooden before the regular one and leave the others where they are.